### PR TITLE
New Feature: scheduler extension, update v2  (#277)

### PR DESCRIPTION
**Pull request by cehbrecht**
# Overview
This PR adds an optional extension to PyWPS to delegate job execution to a batch scheduler system
like Slurm, Grid Engine and Torque.

Extra requirements are in requirements-processing.txt and documentation has been updated for this extension.

# Related Issue / Discussion
See feature request #277

# Additional Information
It is a first working version of this extension. Needs to be adapted to PyWPS needs (Celery).

This is an updated version of PR #278. 
A unit test for the new processing module has been added. 

See Gitter Chat from 24.08.2017.

# Contribution Agreement

(as per https://github.com/geopython/pywps/blob/master/CONTRIBUTING.rst#contributions-and-licensing)

- [ ] I'd like to contribute [feature X|bugfix Y|docs|something else] to PyWPS. I confirm that my contributions to PyWPS will be compatible with the PyWPS license guidelines at the time of contribution.
- [x] I have already previously agreed to the PyWPS Contributions and Licensing Guidelines

### CODE REVIEW

File: tests/__init__.py [hunk 1]
```diff
@@ -14,10 +14,11 @@
 from tests import test_inout
 from tests import test_literaltypes
 from tests import validator
-from tests import test_ows
```
Comment (jorgejesus) [inline tests/__init__.py]: Why is the test_ows disabled?

Comment (cehbrecht) [reply tests/__init__.py]: oops ... I checked it on my side and it must have happend during development ... test_ows failed without gdal ... didn't notice.

I've enabled these tests again and they pass (with gdal). 

Should I open a new PR or is this acceptable?

Comment (jorgejesus) [reply tests/__init__.py]: leave it as it is I also tested and things where ok, later will patch

File: docs/extensions.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+install the dependencies with::
+
+  $ conda install drmaa dill
+
+The package `dill`_ is an enhanced version
+of the Python pickle module for serializing and de-serializing Python objects.
+
+.. warning:: In addition you need to install and configure the drmaa modules for
+  your scheduler system on the machine PyWPS is running on. Follow the
+  instructions given in the `DRMAA`_ documentation and by your scheduler system
```
Comment (jorgejesus) [inline docs/extensions.rst]: LOL been looking in internet how to do it not nothing clear

**Comment by tomLandry** [comment]
This is great news. We will be early users of this feature for sure. We're next in line with a Celery-based sceduler for PyWPS!

**Pull request merged**